rcldotnet_utils/rcldotnet_utils/unity_files/start_player.py

```diff
@@ -0,0 +1,26 @@
+import os
+import sys
+import subprocess
+import platform
+
+build_directory = os.path.dirname(os.path.realpath(sys.argv[0]))
+
+if platform.system() == 'Windows':
+    library_path = build_directory + '\\BuiltGame_Data\\Plugins'
```

Comment:
I have some issues with the "Game" here, since we are both more rather making a "Simulation"

Reply:
Yea, that was just the default name for the build. Thought about making it possible to set it to what you want in a dialogue, but that would require the C# code to modify or generate the python script that starts the app.

Changing it to "Simulation" makes sense.